common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AccessCondition.scala

```diff
@@ -0,0 +1,24 @@
+package uk.ac.wellcome.models.work.internal
+
+import java.time.LocalDate
+
+case class AccessCondition(
+  status: AccessStatus,
+  terms: Option[String],
+  to: Option[LocalDate]
```

Comment:
Should we keep this as an Instant and render it the same as all other dates that we're using? I'm not sure why we would need to introduce another Date type.

Reply:
The reasons I would introduce an extra type is that `LocalDate` seems to better represent the data:

1) We are just interested in a day rather than hours, minutes, seconds, ms, microseconds etc
2) There doesn't appear to be timezone information encoded within the date

Reply:
@jamesgorrie any thoughts on above? :point_up:

Reply:
> We are just interested in a day rather than hours, minutes, seconds, ms, microseconds etc 

For now until we need more resolution.

> There doesn't appear to be timezone information encoded within the date

It will almost definitely be GMT / UTC.

It represents the `Display` logic of the date, I'd opt for `Instant`s where we can unless we can really explain why we'd use something else except displaying, which we have other ways of dealing with that (store as instant, display as whatever).

Reply:
Hmmm personally I would err on the side of modelling with the more accurate representation without any redundancy (according to our current knowledge), but will convert to `Instant` to be consistent with the dates elsewhere.

Reply:
I have updated it to use `Instant` instead of `LocalDate`, but feels a bit wonky to me if we are serialising to the front end as solely a date / expecting the incoming data to the transformer to be a date rather than an instant.

https://github.com/wellcometrust/catalogue/pull/335/commits/6dc1edc4817d663cda6eb3908161f2d852159afb